starter-core/src/main/resources/pom.xml

```diff
@@ -25,17 +25,17 @@
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-noarg</artifactId>
-            <version>1.4.31</version>
+            <version>1.4.32</version>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-allopen</artifactId>
-            <version>1.4.31</version>
+            <version>1.4.32</version>
         </dependency>
         <dependency>
             <groupId>org.jetbrains.kotlin</groupId>
             <artifactId>kotlin-gradle-plugin</artifactId>
-            <version>1.4.31</version>
+            <version>1.4.32</version>
         </dependency>
         <dependency>
             <groupId>io.micronaut.gradle</groupId>
```

Comment:
Latest version of the micronaut-gradle-plugin is 1.4.6.

Reply:
I have set [micronaut gradle plugin](https://plugins.gradle.org/plugin/io.micronaut.library) to 1.4.6